applications/crossbar/test/cb_alerts_tests.erl

```diff
@@ -0,0 +1,286 @@
+-module(cb_alerts_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+
+%%%=======================================================================================
+%%% Tests generator
+%%%=======================================================================================
+
+%%--------------------------------------------------------------------
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec cb_alerts_test_() -> [{string(), boolean()}].
+cb_alerts_test_() ->
+    check_port_request_status()
+    ++ check_port_request_comments()
+    ++ check_low_balance().
+
+%%%=======================================================================================
+%%% Tests
+%%%=======================================================================================
+
+%%--------------------------------------------------------------------
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec check_port_request_status() -> [{string(), boolean()}].
+check_port_request_status() ->
+    Mod = 'knm_port_request',
+    meck:new(Mod, [passthrough]),
+
+    Context = cb_context:new(),
+
+    %% 1 submitted, 1 scheduled, and 1 rejected.
+    meck:expect(Mod, 'account_active_ports', fun(_) -> account_active_ports() end),
+    Context1 = cb_alerts:check_port_request_status(cb_context:new()),
+
+    %% Not active ports found.
+    meck:expect(Mod, 'account_active_ports', fun(_) -> {'error', 'not_found'} end),
+    Context2 = cb_alerts:check_port_request_status(Context),
+
+    %% Ports without scheduled or rejected state.
+    meck:expect(Mod, 'account_active_ports', fun(_) -> {'ok', [example_port_request()]} end),
+    Context3 = cb_alerts:check_port_request_status(Context),
+
+    meck:unload(Mod),
```

Comment:
I like the way you use `meck`. Would you please add `meck:validate/1` to  be part of the below tests fixtures so eunit will rin it as test? I think for this you have to use setup and teardown test fixtures. For example look into `teletype_publish_tests`.

Another way which I'm not requesting as change request for this PR, but It would be fun to experiment with it, is that you can start kazoo fixturedb and put all your docs in fixturedb and then just run `cb_alerts:validate/*`.